packages/material-ui/src/MobileStepper/MobileStepper.test.js

```diff
@@ -40,92 +46,105 @@ describe('<MobileStepper />', () => {
   }));
 
   it('should render a Paper with 0 elevation', () => {
-    const wrapper = mount(<MobileStepper {...defaultProps} />);
-    expect(wrapper.find(Paper).props().elevation).to.equal(0);
+    const { container } = render(<MobileStepper {...defaultProps} />);
+    const paperClasses = getClasses(<Paper elevation={0} />);
+    expect(container.firstChild).to.have.class(paperClasses.elevation0);
```

Comment:
the feedback here applies or all the other test cases

Reply:
Got it

Reply:
Done!